grc/upload/__init__.py

```diff
@@ -34,7 +35,12 @@ def __init__(self, url: str, data_section: str, html_file: str, file_list: Calla
 ]
 
 def delete_file(application_data, file_name, section):
-    AwsS3Client().delete_object(file_name)
+    try:
+        AwsS3Client().delete_object(file_name)
+    except:
```

Comment:
Can you also print the exception here? i.e.
`except Exception as e:`
`.   logger.log(LogLevel.ERROR, f"Could not delete file {file_name} - {e}")`

Reply:
Good idea - I'll add that 👍